packages/restapi/src/lib/payloads/helpers.ts

```diff
@@ -206,6 +225,26 @@ export async function getVerificationProof({
     message = {
       data: `2+${payloadJSON}`,
     };
+
+    if (chatId !== undefined) {
```

Comment:
Rather than adding check for chatId in only identityType 2
Declare senderType
```
senderType {
channel : 0
w2w: 1
}

```
Keep default senderType as 0 so we don't break anything
and if senderType is 1 we perform pgpv2 verification proof